app/lib/linters/spectral.rb

```diff
@@ -0,0 +1,15 @@
+module Linters
+  class Spectral
+    def initialize(file:)
+      @file = file
+    end
+
+    def lint_to_json
+      `spectral lint -f json "#{file.path}"`
+    end
+
+    private
+
+    attr_reader :file
+  end
+end
```

Comment:
Blank line at end of file need!

Reply:
I need to add the editorconfig in, will do that in this pr!